.github/workflows/ci.yml

```
@@ -157,75 +159,113 @@
+          parallel-finished: true
+          coverage-reporter-version: v0.6.15
+          fail-on-error: false
+
       - name: Download APK Artifact
         uses: actions/download-artifact@v5
         with:
-          name: android-apk-${{ github.run_number }}
-          path: build/app/outputs/apk/release/  
+          name: android-apk
+          path: build/app/outputs/flutter-apk/
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)